src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Colors getRandomColor() {
+        return Colors.values()[new Random().nextInt(Colors.values().length)];
```

Comment:
```suggestion
    private final Random random = new Random();
    
    public Colors getRandomColor() {
        int colorIndex = random.nextInt(Colors.values().length);
        return Colors.values()[colorIndex];
```
Random should be a class-field